OpenEdXMobile/res/values/styles.xml

```diff
@@ -76,7 +76,7 @@
     <style name="AppTheme.Dialog.Alert" parent="Theme.AppCompat.Light.Dialog.Alert">
         <item name="colorPrimary">@color/edx_brand_primary_base</item>
         <item name="colorPrimaryDark">@color/edx_brand_primary_dark</item>
-        <item name="colorAccent">@color/edx_brand_primary_accent</item>
+        <item name="colorAccent">@color/dialog_button_text</item>
```

Comment:
Following the current convention in our project, it would be better to either suffix or prefix the selector's name with `selector` e.g. `dialog_button_selector` or `selector_dialog_button`.